package.json

```diff
@@ -29,21 +29,22 @@
     "generate:contributors": "node ./scripts/updateContributors.mjs icons/*.svg",
     "generate:nextJSAliases": "node ./scripts/generateNextJSAliases.mjs",
     "postinstall": "husky install",
-    "lint:js": "eslint --ext .ts,.js,.mjs ./{packages/lucide,scripts}",
```

Comment:
Now the :js name does not make sense anymore.

Reply:
true, i've renamed it to `lint:es` 😛